docs/topics/DatabaseOperations.adoc

```diff
@@ -316,12 +366,17 @@ ykpersonalize -2 -a -ochal-resp -ochal-hmac -ohmac-lt64 -oserial-api-visible -oa
 ```
 
 You will be asked to enter the HMAC key you created earlier, copy/paste they key output in the first step. Repeat both steps for your second YubiKey. We recommend storing your HMAC key in a safe place (e.g., printed on paper) in case you need to recreate another key.
+
+== Command Line Tool
+KeePassXC comes with a robust command line tool *keepassxc-cli* to access, view, and manipulate your database directly from any terminal window. The tool is separately documented through it's man page which can be shown using `man keepassxc-cli` or through the on-demand help using `keepassxc-cli [command] -h`. An online version of the man page is accessible https://github.com/keepassxreboot/keepassxc/blob/master/docs/man/keepassxc-cli.1.adoc[here].

```

Comment:
"it's man page" -> "its man page".

Reply:
Possessive is correct here. Although I think we should say "the man page"